### DISCUSSION TITLE
不研究安全分析的前提下，只求在流程上和代码上理解 STARK & ALI & FRI，需要对 RScode 掌握多少才够？

### DISCUSSION THREAD

**Discussion by keroro520** [coset-io/zkp-academy#50] [General]
不少资料都引用了 RScode，看得有点懵，但是除了安全分析，又不知道 RScode 还涉及到什么方面。

可以的话，希望能从 ALI 算术化，FRI commitment & query phase 的各个环节来科普一下。

**Comment by HaroldGin931** [comment]
我觉得是个好问题 我最近在整理 RScode 可以先来讨论一下 然后我们再看看讨论下什么内容

**Reply by HaroldGin931** [reply]
这个是笔记的地址 https://github.com/Antalpha-Labs/plonky3-python-notebook/blob/345290d06b47493ffa9630d16a299003b19fec5c/ReedSolomonCode/rscode.md
